nginx/production/nginx.conf

```diff
@@ -51,7 +51,14 @@ http {
         } # managed by Certbot
 
         location / {
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header Host $host;
+
             proxy_pass http://rhizone_lms_webapp;
+
+            proxy_http_version 1.1;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "upgrade";
```

Comment:
The web server that handles requests for `rhi.zone` doesn't accept WebSocket connections, so this configuration is not necessary.

Reply:
Okay, so I only need to remove the config lines I added for the production NGINX for the webserver? Everything else in the dev is fine?